src/pages/billing/CreateNewBillingProjectWizard.test.js

```diff
@@ -0,0 +1,465 @@
+import '@testing-library/jest-dom'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { act } from 'react-dom/test-utils'
+import { h } from 'react-hyperscript-helpers'
+import { Ajax } from 'src/libs/ajax'
+import * as Preferences from 'src/libs/prefs'
+import CreateNewBillingProjectWizard, { styles } from 'src/pages/billing/CreateNewBillingProjectWizard'
+
+
+jest.mock('src/libs/ajax')
+jest.spyOn(Preferences, 'getLocalPref')
+const authorizeAndLoadAccounts = jest.fn()
+
+const getStep1Button = () => screen.getByText('Go to Google Cloud Console')
+const getStep2DontHaveBillingAccountButton = () => screen.getByText("I don't have access to a Cloud billing account")
+const getStep2HaveBillingAccountButton = () => screen.getByText('I have a billing account')
+const getStep3DontHaveAccessButton = () => screen.queryByLabelText("I don't have access to do this")
+const getStep3HaveAddedButton = () => screen.queryByLabelText('I have added terra-billing as a billing account user (requires reauthentication)')
+const getStep3CheckBox = () => screen.queryByRole('checkbox')
```

Comment:
Is it possible to get this by the text? It would be better because it would verify the text and also not rely on being the only checkbox on the page.